pkg/menu/menu.go

```diff
@@ -38,6 +38,16 @@ func max(a, b int) int {
 	return b
 }
 
+func countNewlines(str string) int {
+	count := 0
+	for i := 0; i < len(str); i++ {
+		if str[i:i+1] == "\n" {
+			count++
+		}
+	}
+	return count
+}
```

Comment:
another way to count newlines
len(strings.Split(b[:i], "\n"))-1

https://play.golang.org/p/nuX1p_BGX7a

but in any event  use an iterator
for _, s := range str {
if s == '\n' {
count++
}
}

Reply:
Sounds good, I'll probably use the iterator version just for clarity.

Reply:
ok but def write a test for it then to cover 0 length, > 0 with no newline, only newline, etc.